cmake/Modules/DefineInternalVariables.cmake

```diff
@@ -35,7 +35,15 @@
 
 # Define other internal variables
 #
-set(KIM_API_STANDARD_INSTALL_PREFIXES "/" "/usr" "/usr/local" CACHE INTERNAL "Cononical list of standard install prefixes")
+if(WIN32 AND NOT CYGWIN)
+  # Use ';' as path list separator on native Windows platform (including MinGW).
+  set(KIM_API_PATH_SEPARATOR ";" CACHE INTERNAL "Canonical separator character used in path lists")
+  set(KIM_API_STANDARD_INSTALL_PREFIXES "c:/Program Files/${PROJECT_NAME}" CACHE INTERNAL "Canonical list of standard install prefixes")
```

Comment:
I chose lowercase `c:` to match the default value of `CMAKE_INSTALL_PREFIX` on Windows, [see here](https://cmake.org/cmake/help/latest/variable/CMAKE_INSTALL_PREFIX.html).

Reply:
OK that's good.  In that case, there is a `C:/` in `INSTALL` line 193 that we can change to `c:/`

Reply:
I would recommend leaving C: in `INSTALL` capitalized, because it is the more common way of writing it. Having lowercase c: in `KIM_API_STANDARD_INSTALL_PREFIXES` is needed to support case-sensitive string comparison with `CMAKE_INSTALL_PREFIX` in `utils/CMakeLists.txt`, I think.

Reply:
OK.  Reasonable.